### PR TITLE
Support multi-user environment in VSCode tasks

### PR DESCRIPTION
**Pull request by vitalybu**

### CODE REVIEW

**Review by morantr — APPROVED** [review]

File: .vscode/launch.json [hunk 1]
```diff
@@ -64,7 +64,7 @@
                 "pipeArgs": [
                     "exec",
                     "-i",
-                    "gateway-root",
+                    "gateway-${env:USER}",
```
Comment (tomereli) [inline .vscode/launch.json:67]: Note that this will not work if we run vscode as sudo, and then it needs to be `"gateway-${env:SUDO_USER}"`. But I don't know how to specify this in the launch.json file, so OK.

File: .vscode/tasks.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
         "command": "rm",
         "args": [
           "-rf",
-          "build"
+          "build/*"
```
Comment (tomereli) [inline .vscode/tasks.json:35]: Why not delete the full build folder than? Should be a separate commit (nitpick)

**Review by tomereli — APPROVED** [review]

File: .vscode/tasks.json (hunk 2 above)
Comment (vitalybu) [reply .vscode/tasks.json:35]: In case you want to link the `build` directory to a different location (e.g. ramdisk to speed up the build).

File: .vscode/launch.json (hunk 1 above)
Comment (vitalybu) [reply .vscode/launch.json:67]: Why would you run vscode as sudo?
